pnpm-lock.yaml

```diff
@@ -104,258 +103,446 @@ importers:
         version: 5.4.5
 
 packages:
-
   '@aashutoshrathi/word-wrap@1.2.6':
-    resolution: {integrity: sha512-1Yjs2SvM8TflER/OD3cOjhWWOZb58A2t7wpE2S9XfBYTiIl+XFhQG2bjy4Pu1I+EAlCNUzRDYDdFwFYUKvXcIA==}
-    engines: {node: '>=0.10.0'}
+    resolution:
+      {
+        integrity: sha512-1Yjs2SvM8TflER/OD3cOjhWWOZb58A2t7wpE2S9XfBYTiIl+XFhQG2bjy4Pu1I+EAlCNUzRDYDdFwFYUKvXcIA==,
+      }
+    engines: { node: '>=0.10.0' }
```

Comment:
ah it seems like this file has been formatted by Prettier or something, we can probably revert the changes here and then re-run `pnpm install` to generate a minimal diff

Reply:
8f434da2b917ee5829ef2b7b174573b4fc75e7f6
3e3ecd5a3e0d5d47ddedb9597ec3b76eeffa1029